csv-validator-cmd/src/test/resources/uk/gov/nationalarchives/csv/validator/cmd/integrityCheck/WO_95/tech_acq_metadata_v1_WO95Y14B000.csvs

```diff
@@ -1,5 +1,5 @@
 version 1.0
-@totalColumns 33
+@totalColumns 33 @integrityCheck("file_path","excludeFolder")
```

Comment:
The integrity check is not enforced by default. If there is no global directive it will behave like before.